sdk/src/accounts/webSocketProgramAccountSubscriberV2.ts

```diff
@@ -51,6 +71,16 @@ export class WebSocketProgramAccountSubscriberV2<T>
 	private accountsCurrentlyPolling: Set<string> = new Set(); // Track which accounts are being polled
 	private batchPollingTimeout?: ReturnType<typeof setTimeout>; // Single timeout for batch polling
 
+	// For batching initial account fetches
+	private accountsPendingInitialMonitorFetch: Set<string> = new Set(); // Track accounts waiting for initial monitor fetch
```

Comment:
without much docs on this, it really confused me a lot, thinking why we are doing the fallback polling at the beginning